xml_converter/src/xml_converter.cpp

```diff
@@ -446,8 +372,36 @@ vector<string> get_xml_files(string directory, string data_directory) {
     return files;
 }
 
-void convert_taco_directory(string directory, string data_directory, map<string, Category>* marker_categories, vector<Parseable*>* parsed_pois) {
-    vector<string> xml_files = get_xml_files(directory, data_directory);
+void move_supplimentary_files(string input_directory, string output_directory) {
+    DIR* dir = opendir(input_directory.c_str());
+    struct dirent* entry;
+    while ((entry = readdir(dir)) != NULL) {
+        string filename = entry->d_name;
+        if (filename != "." && filename != "..") {
+            string path = input_directory + "/" + filename;
+            if (entry->d_type == DT_DIR) {
+                string new_directory = output_directory + "/" + filename;
+                if (mkdir(new_directory.c_str(), 0700) == -1 && errno != EEXIST) {
+                    cout << "Error making " << new_directory << endl;
+                    continue;
+                }
+                move_supplimentary_files(path, new_directory);
+            }
+            else if (has_suffix(filename, ".trl") || has_suffix(filename, ".xml")) {
+                continue;
+            }
+            else {
+                // TODO: Only include files that have references
```

Comment:
Which references?

This might be obvious now but it wont necessarily be obvious in the future that you are talking about "files that are referenced from the marker data"